src/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.Bignum.cs

```diff
@@ -3,33 +3,23 @@
 
 using System;
 using System.Runtime.InteropServices;
-
 using Microsoft.Win32.SafeHandles;
 
 internal static partial class Interop
 {
-    internal static partial class libcrypto
+    internal static partial class Crypto
     {
-        [DllImport(Libraries.LibCrypto)]
-        internal static extern void BN_clear_free(IntPtr a);
-
-        [DllImport(Libraries.LibCrypto)]
-        private static extern IntPtr BN_bin2bn(byte[] s, int len, IntPtr zero);
+        [DllImport(Libraries.CryptoNative)]
+        internal static extern void BigNumDestroy(IntPtr a);
 
-        [DllImport(Libraries.LibCrypto)]
-        private static extern unsafe int BN_bn2bin(SafeBignumHandle a, byte* to);
+        [DllImport(Libraries.CryptoNative)]
+        private static extern IntPtr BigNumFromBinary(byte[] s, int len);
```

Comment:
This method is called from `internal static IntPtr CreateBignumPtr` below. CreateBignumPtr is called from https://github.com/dotnet/corefx/blob/master/src/System.Security.Cryptography.OpenSsl/src/System/Security/Cryptography/RSAOpenSsl.cs#L291 and put on the "RSA_ST" struct.  The RSA_ST struct is eventually freed by SafeRsaHandle, which calls [RSA_free](http://openssl.org/docs/manmaster/crypto/RSA_new.html) which 
>RSA_free() frees the RSA structure and its components.

This will take care of freeing the Bignum memory.